handler/src/main/java/io/netty/handler/ssl/OpenSslExtendedKeyMaterialManager.java

```diff
@@ -28,12 +28,13 @@
     }
 
     @Override
-    protected String chooseClientAlias(OpenSslEngine engine, String[] keyTypes, X500Principal[] issuer) {
+    protected String chooseClientAlias(ReferenceCountedOpenSslEngine engine, String[] keyTypes,
+                                       X500Principal[] issuer) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Consider using varargs for methods or constructors which take an array the last parameter. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=pmd%3AUseVarargs)